packages/keyring-snap-bridge/src/SnapKeyring.test.ts

```diff
@@ -427,11 +451,13 @@ describe('SnapKeyring', () => {
             keyring = new SnapKeyring(mockSnapKeyringMessenger, mockCallbacks);
 
             const params = {
-              account: account as unknown as KeyringAccount,
-              ...(displayConfirmation !== undefined && { displayConfirmation }),
+              account,
               ...(accountNameSuggestion !== undefined && {
                 accountNameSuggestion,
               }),
+              // Those flags have now been deprecated, but some older Snaps might still use them, so we
+              // must be able to use them in the `AccountCreated` without throwing any error.
```

Comment:
Are they used if set?

Reply:
Nope, not anymore. I was wondering if that was useful or not...